indico/modules/events/abstracts/templates/management/overview.html

```diff
@@ -94,9 +94,17 @@
                 <div class="label">
                     {%- trans %}Book of abstracts{% endtrans -%}
                 </div>
-                {%- trans url=url_for('.export_boa', event) -%}
-                    Customize and <a href={{ url }}>download</a> the book of abstracts
+                {%- trans -%}
+                    Customize and <a class="arrow js-dropdown" data-toggle="dropdown">download</a> the book of abstracts
```

Comment:
I think it's time to use `{{ link_start }}` and `{{ link_end }}` in here - it's quite ugly to have that much HTML in the translatable string. If you search for `{% set link_start %}` you'll find some examples.